### PR TITLE
Es6 syntax

### PR DESCRIPTION
**Pull request by frank1738**
**In this milestone, the following was done:**
- created ES6 module files.
- Added ES6 syntax to JS files
- Added HTML and CSS files
- Added linters file to lint my files

### CODE REVIEW

File: modules/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  e.preventDefault();
+  if (bookTitle.value === '' || bookAuthor.value === '') {
+    return;
+  }
+  myLibrary.addBook(bookTitle, bookAuthor);
+  localStorage.setItem('book-collection', JSON.stringify(myLibrary.library));
+  bookTitle.value = '';
+  bookAuthor.value = '';
+  window.location.reload();
+};
```
Comment (Gambit142) [inline modules/AddBook.js:16]: - [ ] (Optional) Remember that this application is meant to be a Single Page Application (SPA). Using `window.relocation.reload()` after adding a book will defeat the purpose. I would advise you to either toggle the classlist or set the add book section to display none and the list book section to display block. 👍

File: modules/Library.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// import { bookSection } from '../index.js';
```
Comment (Gambit142) [inline modules/Library.js:1]: - [ ] Kindly remove all unnecessary codes from the file. Doing so makes your code clean. 👍

File: modules/SetTime.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/prefer-default-export *//* eslint-disable import/prefer-default-export */
+import { DateTime } from './luxon/time.js';
+
+const siteDate = document.querySelector('.time');
+export const currentTime = () => {
+  const theTime = DateTime.now();
+  siteDate.innerHTML = `${theTime.c.day}/${theTime.c.month}/${theTime.c.year}`;
+};
```
Comment (Gambit142) [inline modules/SetTime.js:8]: - [ ] Kindly display the time and date and not only the date. You can format it by doing this ` const theTime = DateTime.now().toFormat("ff' UTC'Z")` or any other format you want to use but please display both the time and date. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @frank1738 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- All linters checks passed 💯 
- Great work using modular Javascript 💯 

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- You can also consider the comments under the review

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Emmyn5600 — APPROVED** [review]
Hi @frank1738 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [ ] N/A

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**